WcaOnRails/.rubocop.yml

```diff
@@ -54,3 +54,9 @@ Style/SignalException:
 Style/ClassVars:
   Exclude:
     - app/models/country.rb
+
+Style/IfInsideElse:
+  Enabled: false
+
+Lint/NonLocalExitFromIterator:
+  Enabled: false
```

Comment:
Disabled because I can't think of a better way of writing: https://github.com/jfly/worldcubeassociation.org/blob/5868aef053809fdcf9d01776e7208b8e3d988cd0/WcaOnRails/app/models/result.rb#L40.